MemberExpectations.md

```diff
@@ -0,0 +1,58 @@
+It is important that members of our leadership groups (TSC, CommComm) act
+in a way that not only complies with the code of conduct but that supports
+the vision, mission and operating principles of the Node.js project.
+
+While members have the right and responsibility to exercise independent
+judgment and to express dissenting opinions during discussions within
+the organization, members also have the obligation outside the
+organization (twitter, reddit, etc.) to respect and support decisions
+and values of the majority, even when a member dissented from
+the majority view. A member who does not support a team decision
+or current team value may express the member's opposition within
+the team in an appropriate manner, but must not take actions publicly
+that have the purpose or result of undermining the decisions, values
+or actions of the team. Accordingly, a member who itends to publicly
+oppose a team decision action should resign from the team before doing so.
+
+In addition to being willing to act as a team player, members of our
+leadership groups must conduct themselves in a professional and
+respectful manner. Some general
+guidelines include:
+
+- Serve as ambassadors of the vision, mission and operating
+  principles of the Node.js project.
+- Treat all community members as professionals. with respect, consideration,
```

Comment:
Comma after professionals?

Reply:
Or maybe a colon?:

> Treat all community members as professionals: with respect, consideration, and using the highest standards of ethical conduct.

Another option is to simplify it:

> Treat all community members with respect, consideration, and the utmost ethical conduct.